src/util.cpp

```diff
@@ -1705,31 +1722,38 @@ QString GetVersionAndNameStr ( const bool bDisplayInGui )
         strVersionText += "\n *** Foundation; either version 2 of the License, or (at your option) any later version.";
         strVersionText += "\n *** There is NO WARRANTY, to the extent permitted by law.";
         strVersionText += "\n *** ";
-        strVersionText += "\n *** Using the following libraries, resources or code snippets:";
+
+        strVersionText += "\n *** " + QCoreApplication::tr ( "This app uses the following libraries, resources or code snippets:" );
```

Comment:
Feels a bit like code duplication...

Reply:
Yes - different issue, though.

Keep it small and simple.